dev/reference/sql/functions-and-operators/expressions-pushed-down.md

```diff
@@ -0,0 +1,147 @@
+---
+title: List of Operators for Pushdown
+summary: This document introduces a list of operators that can be pushed down to TiKV, the related operations of which are also provided.
+category: reference
+---
+
+# List of Operators for Pushdown
+
+When TiDB reads data from TiKV, TiDB tries to push down some expressions to TiKV to be processed thus reducing the amount of data transferred and the computational pressure on a single TiDB node. This document introduces operators that TiDB supports for pushdown and how to blacklist specific operators.
```

Comment:
Can the `pushdown` be used as a noun？
This document introduces which TiDB operators can be pushed down to TiKV, and how to disable this function by putting specific operators into a blacklist.

Reply:
> Can the `pushdown` be used as a noun？
> This document introduces which TiDB operators can be pushed down to TiKV, and how to disable this function by putting specific operators into a blacklist.

Thanks for your sharp eyes! I have done a little search actually, and to answer your question, yes, `pushdown` can be used as a noun, but with a totally different meaning. However, in [one of the blogs that we have posted](https://pingcap.com/blog/adding-built-in-functions-to-tikv/), we used expressions like "select a function for pushdown", so it is justifiable to use `pushdown` here. Similarily, "pushdown" can also be used as an adjective.
Also, apologies for my late follow-up.